src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -265,7 +267,8 @@ public virtual IEnumerable<PackageResult> List(ChocolateyConfiguration config)
                                 !string.IsNullOrWhiteSpace(package.BugTrackerUrl.ToStringSafe()) ? "{0} Issues: {1}".FormatWith(Environment.NewLine, package.BugTrackerUrl.ToStringSafe()) : string.Empty,
                                 package.Summary != null && !string.IsNullOrWhiteSpace(package.Summary.ToStringSafe()) ? "\r\n Summary: {0}".FormatWith(package.Summary.EscapeCurlyBraces().ToStringSafe()) : string.Empty,
                                 package.Description.EscapeCurlyBraces().Replace("\n    ", "\n").Replace("\n", "\n  "),
-                                !string.IsNullOrWhiteSpace(package.ReleaseNotes.ToStringSafe()) ? "{0} Release Notes: {1}".FormatWith(Environment.NewLine, package.ReleaseNotes.EscapeCurlyBraces().Replace("\n    ", "\n").Replace("\n", "\n  ")) : string.Empty
+                                !string.IsNullOrWhiteSpace(package.ReleaseNotes.ToStringSafe()) ? "{0} Release Notes: {1}".FormatWith(Environment.NewLine, package.ReleaseNotes.EscapeCurlyBraces().Replace("\n    ", "\n").Replace("\n", "\n  ")) : string.Empty,
+                                !string.IsNullOrWhiteSpace(softwareInstallLocation) ? "{0} Software installed to: '{1}'".FormatWith(Environment.NewLine, softwareInstallLocation) : string.Empty

```

Comment:
```suggestion
                                !string.IsNullOrWhiteSpace(softwareInstallLocation) ? "{0} Software Install Location: '{1}'".FormatWith(Environment.NewLine, softwareInstallLocation) : string.Empty

```

Reply:
Based on a conversation with @pauby we are going to change this to `Deployed to`